src/widgets/search-box/search-box.js

```diff
@@ -24,108 +24,23 @@ const renderer = ({
   showReset,
   showSubmit,
   showLoadingIndicator,
-}) => (
-  { refine, clear, query, onHistoryChange, isSearchStalled },
-  isFirstRendering
-) => {
-  if (isFirstRendering) {
-    const input = document.createElement('input');
-    const wrappedInput = wrapInputFn(input, cssClasses);
-    containerNode.appendChild(wrappedInput);
-
-    if (showSubmit) {
-      addSubmit(input, cssClasses, templates);
-    }
-    if (showReset) {
-      addReset(input, cssClasses, templates, clear);
-    }
-    if (showLoadingIndicator) {
-      addLoadingIndicator(input, cssClasses, templates);
-    }
-
-    addDefaultAttributesToInput(placeholder, input, query, cssClasses);
-
-    // When the page is coming from BFCache
-    // (https://developer.mozilla.org/en-US/docs/Working_with_BFCache)
-    // then we force the input value to be the current query
-    // Otherwise, this happens:
-    // - <input> autocomplete = off (default)
-    // - search $query
-    // - navigate away
-    // - use back button
-    // - input query is empty (because <input> autocomplete = off)
-    window.addEventListener('pageshow', () => {
```

Comment:
this compatibility layer has been removed

Reply:
I wasn't able to reproduce. Were you?

Reply:
This was something @bobylito found a while ago only in FireFox, but I'm not too sure when it happens

Reply:
I tried to follow the steps in Firefox 64.0.2 but doesn't seem to do anything either.